[Bug] Crashing when entering end dimension

### Bug Description

Crashed every time while entering the end within my modpack, found out it was this mod; to double check i created a new modpack with only this mod installed and it still happened. I'm not familiar with github so it might be an issue on my side, if that is so apologies.

### How to Reproduce?

Create new modpack on curseforge 1.21.3
add only debugify as a mod
create new world
enter end portal

### Expected Behavior

freezes on loading screen, or crashes on loading screen

### Version

1.21.3

### Fabric/Quilt Loader Version

0.16.9

### Logs or additional context

---- Minecraft Crash Report ----
// You should try our sister game, Minceraft!

Time: 2024-11-29 01:46:56
Description: Exception ticking world

java.lang.ExceptionInInitializerError
	at knot//net.minecraft.class_1510.method_5693(class_1510.java:159)
	at knot//net.minecraft.class_1297.<init>(class_1297.java:323)
	at knot//net.minecraft.class_1309.<init>(class_1309.java:282)
	at knot//net.minecraft.class_1308.<init>(class_1308.java:152)
	at knot//net.minecraft.class_1510.<init>(class_1510.java:104)
	at knot//net.minecraft.class_1299.method_5883(class_1299.java:582)
	at knot//net.minecraft.class_2881.method_12523(class_2881.java:494)
	at knot//net.minecraft.class_2881.method_12525(class_2881.java:305)
	at knot//net.minecraft.class_2881.method_12538(class_2881.java:251)
	at knot//net.minecraft.class_3218.method_18765(class_3218.java:373)
	at knot//net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:1069)
	at knot//net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:953)
	at knot//net.minecraft.class_1132.method_3748(class_1132.java:114)
	at knot//net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:713)
	at knot//net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:292)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_1518 failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:427)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at knot//net.minecraft.class_1527.<clinit>(class_1527.java:13)
	... 16 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422)
	... 21 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Callback method getLandingPos(Lnet/minecraft/class_2338;)Lnet/minecraft/class_2338; in debugify.mixins.json:basic.mc88371.DragonLandingPhaseMixin from mod debugify failed injection check, (0/1) succeeded. Scanned 0 target(s). Using refmap Debugify-refmap.json
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.postInject(InjectionInfo.java:531)
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1490)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:752)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:330)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:246)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:437)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:418)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
	... 24 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at knot//net.minecraft.class_1510.method_5693(class_1510.java:159)
	at knot//net.minecraft.class_1297.<init>(class_1297.java:323)
	at knot//net.minecraft.class_1309.<init>(class_1309.java:282)
	at knot//net.minecraft.class_1308.<init>(class_1308.java:152)
	at knot//net.minecraft.class_1510.<init>(class_1510.java:104)
	at knot//net.minecraft.class_1299.method_5883(class_1299.java:582)
	at knot//net.minecraft.class_2881.method_12523(class_2881.java:494)
	at knot//net.minecraft.class_2881.method_12525(class_2881.java:305)
	at knot//net.minecraft.class_2881.method_12538(class_2881.java:251)

-- Affected level --
Details:
	All players: 1 total; [class_3222['Bruno091004'/24, l='ServerLevel[New World]', x=100.50, y=49.00, z=0.50]]
	Chunk stats: 1681
	Level dimension: minecraft:the_end
	Derived: true
	Level spawn location: World: (48,78,192), Section: (at 0,14,0 in 3,4,12; chunk contains blocks 48,0,192 to 63,255,207), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 794 game time, 794 day time
	Level name: New World
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Commands: true
	Level weather: Rain time: 115635 (now: false), thunder time: 160597 (now: false)
	Known server brands: fabric
	Removed feature flags: 
	Level was modded: true
	Level storage version: 0x04ABD - Anvil
	Loaded entity count: 11
Stacktrace:
	at knot//net.minecraft.class_3218.method_8538(class_3218.java:1734)
	at knot//net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:1072)
	at knot//net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:953)
	at knot//net.minecraft.class_1132.method_3748(class_1132.java:114)
	at knot//net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:713)
	at knot//net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:292)
	at java.base/java.lang.Thread.run(Unknown Source)

-- System Details --
Details:
	Minecraft Version: 1.21.3
	Minecraft Version ID: 1.21.3
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 21.0.5, Eclipse Adoptium
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Eclipse Adoptium
	Memory: 928333520 bytes (885 MiB) / 1862270976 bytes (1776 MiB) up to 4294967296 bytes (4096 MiB)
	CPUs: 12
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 5 5600 6-Core Processor              
	Identifier: AuthenticAMD Family 25 Model 33 Stepping 2
	Microarchitecture: Zen 3
	Frequency (GHz): 3.50
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: AMD Radeon RX 6800 XT
	Graphics card #0 vendor: Advanced Micro Devices, Inc.
	Graphics card #0 VRAM (MiB): 16368.00
	Graphics card #0 deviceId: VideoController1
	Graphics card #0 versionInfo: 32.0.12019.1028
	Memory slot #0 capacity (MiB): 16384.00
	Memory slot #0 clockSpeed (GHz): 2.67
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MiB): 16384.00
	Memory slot #1 clockSpeed (GHz): 2.67
	Memory slot #1 type: DDR4
	Virtual memory max (MiB): 37535.09
	Virtual memory used (MiB): 21743.74
	Swap memory total (MiB): 4864.00
	Swap memory used (MiB): 15.05
	Space in storage for jna.tmpdir (MiB): available: 46906.04, total: 1906897.00
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): available: 46906.04, total: 1906897.00
	Space in storage for io.netty.native.workdir (MiB): available: 46906.04, total: 1906897.00
	Space in storage for java.io.tmpdir (MiB): available: 46906.04, total: 1906897.00
	Space in storage for workdir (MiB): available: 46906.04, total: 1906897.00
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx4096m -Xms256m
	Fabric Mods: 
		debugify: Debugify 1.21.3+1.0
		fabric-api: Fabric API 0.110.0+1.21.3
			fabric-api-base: Fabric API Base 0.4.50+119c825640
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.81+fd37071f40
			fabric-biome-api-v1: Fabric Biome API (v1) 14.0.6+fd37071f40
			fabric-block-api-v1: Fabric Block API (v1) 1.0.28+fd37071f40
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.16+fd37071f40
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 2.0.4+fd37071f40
			fabric-client-tags-api-v1: Fabric Client Tags 1.1.24+fd37071f40
			fabric-command-api-v1: Fabric Command API (v1) 1.2.58+f71b366f40
			fabric-command-api-v2: Fabric Command API (v2) 2.2.37+c9d82ab240
			fabric-commands-v0: Fabric Commands (v0) 0.2.75+df3654b340
			fabric-content-registries-v0: Fabric Content Registries (v0) 9.1.8+fd37071f40
			fabric-convention-tags-v1: Fabric Convention Tags 2.1.3+7f945d5b40
			fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.9.3+84ebbb0e40
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.3.3+fd37071f40
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.4.0+dbad41f140
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 21.0.18+fd37071f40
			fabric-dimensions-v1: Fabric Dimensions API (v1) 4.0.7+fd37071f40
			fabric-entity-events-v1: Fabric Entity Events (v1) 2.0.8+825023f840
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 2.0.12+c2c3ac4740
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.59+fd37071f40
			fabric-item-api-v1: Fabric Item API (v1) 11.1.7+fd37071f40
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.1.17+fd37071f40
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.53+fd37071f40
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.51+df3654b340
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.5.1+6da5ef6940
			fabric-loot-api-v2: Fabric Loot API (v2) 3.0.26+3f89f5a540
			fabric-loot-api-v3: Fabric Loot API (v3) 1.0.14+fd37071f40
			fabric-message-api-v1: Fabric Message API (v1) 6.0.21+fd37071f40
			fabric-model-loading-api-v1: Fabric Model Loading API (v1) 3.0.9+fd37071f40
			fabric-networking-api-v1: Fabric Networking API (v1) 4.3.5+fd37071f40
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 17.0.9+fd37071f40
			fabric-particles-v1: Fabric Particles (v1) 4.0.10+fd37071f40
			fabric-recipe-api-v1: Fabric Recipe API (v1) 7.0.4+fd37071f40
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 5.3.0+2758bfbf40
			fabric-renderer-api-v1: Fabric Renderer API (v1) 4.0.5+fd37071f40
			fabric-renderer-indigo: Fabric Renderer - Indigo 1.9.5+fd37071f40
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.54+73761d2e40
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.15+fd37071f40
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.84+df3654b340
			fabric-rendering-v1: Fabric Rendering (v1) 8.0.8+6922831640
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 5.0.8+fd37071f40
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 3.0.6+fd37071f40
			fabric-screen-api-v1: Fabric Screen API (v1) 2.0.34+fd37071f40
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.101+fd37071f40
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.29+fd37071f40
			fabric-transfer-api-v1: Fabric Transfer API (v1) 5.4.3+fd37071f40
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.2.0+54a41b1c40
		fabricloader: Fabric Loader 0.16.9
			mixinextras: MixinExtras 0.4.1
		java: OpenJDK 64-Bit Server VM 21
		minecraft: Minecraft 1.21.3
		yet_another_config_lib_v3: YetAnotherConfigLib 3.6.1+1.21.2-fabric
			com_twelvemonkeys_common_common-image: common-image 3.10.0
			com_twelvemonkeys_common_common-io: common-io 3.10.0
			com_twelvemonkeys_common_common-lang: common-lang 3.10.0
			com_twelvemonkeys_imageio_imageio-core: imageio-core 3.10.0
			com_twelvemonkeys_imageio_imageio-metadata: imageio-metadata 3.10.0
			com_twelvemonkeys_imageio_imageio-webp: imageio-webp 3.10.0
			org_quiltmc_parsers_gson: gson 0.2.1
			org_quiltmc_parsers_json: json 0.2.1
	Server Running: true
	Player Count: 1 / 8; [class_3222['Bruno091004'/24, l='ServerLevel[New World]', x=100.50, y=49.00, z=0.50]]
	Active Data Packs: vanilla, fabric, fabric-convention-tags-v2
	Available Data Packs: fabric, fabric-convention-tags-v2, minecart_improvements, redstone_experiments, trade_rebalance, vanilla, winter_drop
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Stable
	World Seed: 6606984003539665571
	Suppressed Exceptions: ~~NONE~~
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fabric'; Server brand changed to 'fabric'
	Launched Version: fabric-0.16.9-1.21.3

### Has not been reported.

- [X] I made sure this bug hasn't already been reported.

### Is on most update to date version.

- [X] I made sure I am using the most up to date fabric/quilt loader and mod version.

--- a/src/main/java/dev/isxander/debugify/mixins/basic/mc88371/DragonLandingPhaseMixin.java
+++ b/src/main/java/dev/isxander/debugify/mixins/basic/mc88371/DragonLandingPhaseMixin.java
@@ -11,7 +11,7 @@ import org.spongepowered.asm.mixin.injection.At;
 @BugFix(id = "MC-88371", category = FixCategory.BASIC, env = BugFix.Env.SERVER)
 @Mixin(DragonLandingPhase.class)
 public class DragonLandingPhaseMixin {
-    @ModifyExpressionValue(method = "doServerTick", at = @At(value = "INVOKE", target = "Lnet/minecraft/world/level/Level;getHeightmapPos(Lnet/minecraft/world/level/levelgen/Heightmap$Types;Lnet/minecraft/core/BlockPos;)Lnet/minecraft/core/BlockPos;"))
+    @ModifyExpressionValue(method = "doServerTick", at = @At(value = "INVOKE", target = "Lnet/minecraft/server/level/ServerLevel;getHeightmapPos(Lnet/minecraft/world/level/levelgen/Heightmap$Types;Lnet/minecraft/core/BlockPos;)Lnet/minecraft/core/BlockPos;"))
     private BlockPos getLandingPos(BlockPos pos) {
         if (pos.getY() == 0) {
             // average height of portal

